web/src/EducationBenchmarking.Web/ViewModels/SchoolPlanViewModel.cs

```diff
@@ -16,14 +19,159 @@ public SchoolPlanViewModel(School school, Finances finances, int year) : this(sc
         _finances = finances;
     }
 
+    public SchoolPlanViewModel(School school, Finances finances, FinancialPlanResult plan, int year) : this(school)
+    {
+        SelectedYear = year;
+        _plan = plan;
+        _finances = finances;
+    }
+
     public int? SelectedYear { get; set; }
     public string Name => school.Name;
     public string Urn => school.Urn;
-    public string CurrentTotalIncome => $"{_finances.TotalIncome:C}";
-    public string CurrentTotalExpenditure => $"{_finances.TotalExpenditure:C}";
-    public string CurrentTotalTeacherCosts => $"{_finances.TeachingStaffCosts:C}";
-    public string CurrentTotalNumberOfTeachersFte => $"{_finances.TotalNumberOfTeachersFte}";
-    public string CurrentEducationSupportStaffCosts => $"{_finances.EducationSupportStaffCosts:C}";
-    public int CurrentYearEnd => _finances.YearEnd;
-    public bool IsPrimary => _finances.OverallPhase == "Primary";
+    public decimal CurrentTotalIncome
```

Comment:
A simpler way to write these would be;

`public decimal CurrentTotalIncome => _finances?.TotalIncome ?? throw new ArgumentNullException(nameof(_finances));`


Also should use appropriate exception.

Reply:
fixed - amended in latest commit